medicines/README.md

```diff
@@ -1,33 +1,26 @@
+- [Project architecture](./docs/architecture)
+- [Data](./docs/data)
 
-The API pod would contain a lightweight custom HTTP server, written in a language like Go or Rust. It would be stateless so it can scale out with ease. It aggregates data from all three sources and presents a documented API that conforms to the [Open API 3.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md). It would be read-only to start with.
+## Project contents
 
-The Web pod would initially serve static HTML, CSS and JS files for the bulk of the site. Later on, a decision could be made as to whether this needs to be content managed. The search component would slot into the static site and be client-side React.
+- [api](./api) - code for the public-facing API that exposes medicines data
+- [data-normalizer](./data-normalizer) - helper functions to clean input data for the import process
+- [import](./import) - importer to update files
+- [search](./search) - code for the search service, which holds a searchable index for all public files
+- [web](./web) - everything related to the front-end of the site
 
-The importer pod would be scheduled to run weekly by the K8s scheduler. It would also be written in a language like Go or Rust and would read the files exported from Sentinel, validate and process them before storing data and PDFs to Cosmos and Blob Storage respectively, triggering Azure Search to update its indexes. This would be a temporary component that would be deprecated as the management function is migrated here from Sentinel.
+## Contributor guidelines
 
-We may well be able to carve off an MHRA subdomain and manage this in Azure too. The root “A” record would point to the load balancer. We would need to ask GDS to set up “301” redirects for existing URLs.
-
-## Updates
-
-### 6th November 2019
-
-In the spirit of [evolutionary architecture](https://www.thoughtworks.com/insights/blog/microservices-evolutionary-architecture), we’ve updated this design so that the temporary wordpress site will now only be used for uploading Public Assessment Reports (PARs). Only staff with admin capability will be able to access the site and it will only be visible within the MHRA organisation. The “importer pod” will now import PARs from the wordpress API (as well as the SPCs and PILs from the Sentinel batch export).
-
-This has several benefits:
-
-- The Wordpress site would not be accessible by the public
-- Overall security is significantly improved as no external access or login capability required
-- A more consistent user experience
-- No duplication of the surrounding website content
-- A single search index and user experience for all docs (SPC/PIL/PAR)
+See [contributor guidelines](./docs/contributor_guidelines).
```

Comment:
Is it deliberate that we have `./docs/snake_case/kebab-case.md`? I'd expect `kebab-case` for the directories as well as the markdown files.